frontend/src/components/Searchbar/Searchbar.tsx

```diff
@@ -58,51 +73,113 @@ export interface SearchbarProps extends Omit<InputProps, 'onChange'> {
   value?: string
 
   /**
-   * Function to be invoked when the value in the searchbar input changes (but
-   * the search button has not been clicked).
+   * Optional. Function to be invoked when the value in the searchbar input
```

Comment:
tiny nit: not sure we need the word "Optional" everywhere, as it's already shown by the type. plus if we start putting "Optional" for every optional prop, we'd need to do it consistently, e.g. `isExpandable` doesn't have it rn.

Reply:
I guess i personally prefer to be more explicit, but I agree that the type already shows it. I guess more importantly is to explain how the behavior changes if the prop is omitted, haha

Reply:
no strong opinions about this, am ok to not change